docs/upgrade/upgrade-opensource/upgrade-guide-from-5.1-to-5.2/metric-update-5.1-to-5.2.rst

```diff
@@ -55,3 +55,13 @@ The following metrics are renamed in ScyllaDB 5.2
      - scylla_memory_system_unspooled_dirty_bytes
    * - scylla_memory_virtual_dirty_bytes
      - scylla_memory_unspooled_dirty_bytes
+
+Reporting Latencies
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ScyllaDB 5.2 comes with a new approach to reporting latencies, which are reported using histograms and summaries:
+
+* Histograms are reported per node.
+* Summaries are reported per shard and contain P50, P95, and P99 latency.
+
```

Comment:
@amnonh its not clear:
- what was the state before this release
- what user have to do about this change
- examples would be good

Reply:
@amnonh Could you provide me with the missing information specified by @tzach above?

Reply:
Previously it was histogram per shard
users need to update their monitoring stack and the change would be seemless.

Reply:
So it looks like there's nothing else to add here.